### PR TITLE
Devhouse 2013

### PR DESCRIPTION
**Pull request by larsvegas**

### CODE REVIEW

**Comment by Velrok** [comment]
LGTM. But I just looked at the markdown, not the preview.

**Comment by larsvegas** [comment]
merci... whats about your master thesis? :P

2013/10/8 Waldemar notifications@github.com

**Comment by Velrok** [comment]
Almost done ;) .

Am 08.10.2013 um 18:08 schrieb larsvegas notifications@github.com:

File: source/_posts/2013-10-08-devhouse-friday-chillout.markdown [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+categories:
+tags: [devhouse]
+author: Lars Wolff
+---
+
+[{%img right /images/devhouse.gif%}](http://devhousefriday.org/networks/events/show_event.82979)
+
+## ["Out Of Network"](http://en.wikipedia.org/wiki/Run_of_network)
+
+Adcloud will shutdown until the 1st January 2014,
```
Comment (zeisss) [inline source/_posts/2013-10-08-devhouse-friday-chillout.markdown]: Der satz ist komisch ^^

```
1) Adcloud will shutdown the end of the year, ...
2) Adcloud will shutdown Jan 1st, 2014, ...
3) Adcloud will shutdown on the 1st of January 2014, ...
```

But propably @tjg can help here :)

Comment (luebken) [inline source/_posts/2013-10-08-devhouse-friday-chillout.markdown]: Am besten offiziellen Wortlaut verwenden: Adcloud GmbH, acquired in 2011 by Deutsche Post as a startup
company, will cease its operations by the end of 2013. 
http://adcloud.com/en/blog/2013/07/19/deutsche-post-streamlines-structure-its-online-mar/

Comment (zeisss) [inline source/_posts/2013-10-08-devhouse-friday-chillout.markdown]: +1 für Matthias Vorschlag.

**Pull request merged**